lib/archethic/account/mem_tables_loader.ex

```diff
@@ -130,6 +130,9 @@ defmodule Archethic.Account.MemTablesLoader do
 
       {{to, _token_address}, utxo} ->
         TokenLedger.add_unspent_output(to, utxo, timestamp)
+
+      {{to, _token_address, 0}, utxo} ->
```

Comment:
You should remove the previous pattern matching and replace `0` by `_token_id`

Reply:
Yes that makes more sense, I have done that.